bindata/network/multus/002-rbac.yaml

```diff
@@ -63,3 +63,48 @@ subjects:
 - kind: ServiceAccount
   name: multus
   namespace: openshift-multus
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels: 
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  name: multus:view
+rules:
+- apiGroups: ["apiextensions.k8s.io"]
+  resources:
+  - customresourcedefinitions
+  - customresourcedefinitions/status
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups: ["k8s.cni.cncf.io"]
+  resources: ["*"]
```

Comment:
I don't remember all the details of how roles and bindings work, but we need to make sure that this doesn't give project admins the ability to view net-attach-defs in *other* namespaces.

Reply:
I needs to be a Role, not a ClusterRole.